db/migrations/2_create_index_on_history_groups_attempts_id.sql

```diff
@@ -0,0 +1,5 @@
+-- +migrate Up
+ALTER TABLE `history_groups_attempts` ADD INDEX `ID` (`ID`);
+
+-- +migrate Down
+ALTER TABLE `history_groups_attempts` DROP INDEX `ID`;
```

Comment:
I think all these tables can disappear, so if it causes you perf pb, I can start to see if we can do it now.